IHP/Server.hs

```diff
@@ -139,10 +150,10 @@ application request respond = do
         let ?context = requestContext
         frontControllerToWAIApp RootApplication ErrorController.handleNotFound
 
-runServer :: FrameworkConfig -> Application -> IO ()
-runServer FrameworkConfig { environment = Env.Development, appPort } = Warp.runSettings $
+runServer :: (?applicationContext :: ApplicationContext) => FrameworkConfig -> Application -> IO ()
+runServer config@FrameworkConfig { environment = Env.Development, appPort } = Warp.runSettings $
                 Warp.defaultSettings
-                    |> Warp.setBeforeMainLoop (ByteString.putStrLn "Server started")
+                    |> Warp.setBeforeMainLoop (let ?context = config in Log.info "Server started")
```

Comment:
Here we should still stick to the `ByteString.putStrLn`. The dev server [uses this here](https://github.com/digitallyinduced/ihp/blob/master/exe/IHP/IDE/DevServer.hs#L275) to detect that the app server is just about to be started. A custom log formatter or a wrong log level could then break the dev server / live reload here.